CONTRIBUTING.rst

```diff
@@ -45,6 +45,10 @@ Requirements
 
 - Python >=3.9
 - Pipenv
+- PostgreSQL
+
+.. note::
+    Make sure python versions of pip and pipenv match, as otherwise installing the requirements from the Pipfile may fail.
```

Comment:
Maybe you mean that?
```suggestion
    Make sure the versions of pip and pipenv match, as otherwise installing the requirements from the Pipfile may fail.
```

Reply:
@MVrachev No, actually they cannot match due to the different release cycle. I mean the python version they operate with, f.e. pip may have python 3.9 and 3.11, while pipenv may work with 3.10, so the installation will fail and one needs to add 3.10 to pip.

Reply:
Okay, I see.